ci/Dockerfile

```diff
@@ -35,16 +35,14 @@ RUN apk add --no-cache --no-progress --update --upgrade \
         && \
     mv /etc/apk/repositories /etc/apk/repositories.old && \
     apk add --no-cache --update --upgrade \
-        --repository=http://dl-cdn.alpinelinux.org/alpine/v3.11/main \
-        --repository=http://dl-cdn.alpinelinux.org/alpine/v3.11/community \
+        --repository=http://dl-cdn.alpinelinux.org/alpine/v3.12/main \
+        --repository=http://dl-cdn.alpinelinux.org/alpine/v3.12/community \
         lshw \
         && \
-    mv /etc/apk/repositories.old /etc/apk/repositories && \
-    apk add --no-cache --update --upgrade \
-        --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing \
-        cfssl \
+    mv /etc/apk/repositories.old /etc/apk/repositories \
    ;
 
+RUN GOBIN=/bin GO111MODULE=on go get -v github.com/cloudflare/cfssl/cmd/cfssl@v1.6.0 github.com/cloudflare/cfssl/cmd/cfssljson@v1.6.0
```

Comment:
Also not documented in the PR description - is it necessary?

Reply:
When reviewing a PR, it should only be necessary to read the PR description to get the necessary context. Not only does this let the reviewer know which commits were intentional versus accidental, but one cannot expect that a reviewer is going to trawl through all of the commit messages.

My expectation is that PR descriptions contain roughly:
- What is being done
- Why is it being done?
- Why is this the best approach to take?